core/src/main/resources/hudson/model/AbstractItem/confirm-rename_de.properties

```diff
@@ -0,0 +1,25 @@
+# The MIT License
+#
+# Copyright (c) 2018 CloudBees, Inc.
```

Comment:
Just to make sure, do you work for CloudBees? If no, please adjust the header. Otherwise I owe you 🍺